### DISCUSSION TITLE
extend gitlab repository replication on helm charts artifacts

### DISCUSSION THREAD

**Discussion by blka** [goharbor/harbor#15976] [Ideas]
**Is your feature request related to a problem? Please describe.**
in next gitlab release 14.1 helm chart repos will be available. it would be nice to have option to replicate not only images (current) but also charts.
https://gitlab.com/gitlab-org/gitlab/-/issues/18997#note_626387519

currently only harbor<=> harbor chart replication is available
![image](https://user-images.githubusercontent.com/14985010/125775860-e3cabe9e-d342-48c0-bf7a-116abdc69d04.png)

**Describe the solution you'd like**
ability to replicate images AND helm charts from gitlab repos
![image](https://user-images.githubusercontent.com/14985010/125775905-31578692-60dd-4aa0-af35-7bdecddc8bb5.png)

**Additional context**

**Comment by castorls** [comment]
Also a needed feature in our usage. :-)

**Comment by xaleeks** [comment]
Interested in hearing from the community on this scenario.
